### DISCUSSION TITLE
Github Pages not deploying remote theme

### DISCUSSION THREAD

**Discussion by kgarg8** [community/community#21595] [Other Feature Feedback, Questions, & Ideas]
<p>I am trying to use remote theme on Github Pages for the first time. Although the theme works fine on the local server, it is not being deployed on the Github Pages server. I can open the page but the theme is not being loaded correctly.</p>
<p>I already tried the modifications to <code>_config.yml</code> mentioned <a href="https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll" rel="noopener nofollow ugc">here</a>, <a href="https://github.com/benbalter/jekyll-remote-theme" rel="noopener nofollow ugc">here</a>.</p>
<p>Here’s the link to my <a href="https://github.com/kgarg8/Text-Style-Transfer" rel="noopener nofollow ugc">repo</a>. Any help will be much appreciated.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’ve had a look and the theme you’re referencing doesn’t seem to exist. Where are you getting the username/repository reference from?</p>

**Comment by kgarg8** [comment]
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/samarsault/texture" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/30385849/181065764-c956556c-2290-45d2-a9fa-4b26a369db65.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29152b6613b3cfe9ff5e2bb051b53c897999ea29_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29152b6613b3cfe9ff5e2bb051b53c897999ea29_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/9/29152b6613b3cfe9ff5e2bb051b53c897999ea29.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29152b6613b3cfe9ff5e2bb051b53c897999ea29_2_10x10.png"></div>

<h3><a href="https://github.com/samarsault/texture" target="_blank" rel="noopener nofollow ugc">samarsault/texture</a></h3>


  <p><span class="label1">A configurable jekyll theme for simply beautiful blogs.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by canuckjacq** [comment]
<p>In your config file, you have</p>
<pre><code> # Build settings
 remote_theme: thelehhman/texture</code></pre>

**Comment by kgarg8** [comment]
<p>So, I think you meant to indicate:</p>
<p><code>remote_theme: samarsault/texture</code></p>
<p>But this doesn’t solve the problem still.</p>
<p>I used <code>remote_theme: thelehhman/texture</code> since it is mentioned on the same page.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>From the readme:</p>
<blockquote>
<p>For public GitHub, remote themes must be in the form of <code>OWNER/REPOSITORY</code> , and must represent a public GitHub-hosted Jekyll theme.</p>
</blockquote>
<p>So I checked and your config file at the time said:</p>
<pre><code> # Build settings
  remote_theme: thelehhman/texture
</code></pre>
<p>but that username/repository combination doesn’t exist.</p>
<p>It looks like you’ve changed it to</p>
<pre><code> remote_theme: samarsault/texture
</code></pre>
<p>and your Pages site certainly looks themed now.</p>
<p>Let us know if you have any further questions.</p>

**Comment by kgarg8** [comment]
<p>Yeah, the problem was the incorrect info on the README of the repo, got it confirmed from the authors of the theme.</p>
<p>Additionally, I also replaced <code>baseurl: ''```` to </code>baseurl:``` for this to work.</p>
